sdks/v4-sdk/src/utils/abi.ts

```diff
@@ -3,70 +3,188 @@ export const abi = [
   {
     type: 'constructor',
     inputs: [
-      { name: '_poolManager', type: 'address', internalType: 'contract IPoolManager' },
-      { name: '_permit2', type: 'address', internalType: 'contract IAllowanceTransfer' },
-      { name: '_unsubscribeGasLimit', type: 'uint256', internalType: 'uint256' },
-      { name: '_tokenDescriptor', type: 'address', internalType: 'contract IPositionDescriptor' },
+      {
```

Comment:
can we rename this file to be clearer _what_ it is the ABI for? i had to read the ABI to figure it out lol